.idea/gradle.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Do you think it's necessary to include the .idea/gradle.xml on the remote version?

Reply:
No, indeed, I am going to remove it.